layouts/borrowing/list.html

```diff
@@ -1,17 +1,52 @@
 {{ define "main" }}
+<link rel="stylesheet" href="/css/borrowing.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
 <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
 <div class="container">
     <div class="row mb-3 title">
         <h1 class="m-0 title-cn"> {{ .Title }}</h1>
         <p class="title-en" style="margin-left: -4rem">Borrowing</p>
     </div>
     <br />
-    <a href="/file/先鋒13樓科學會辦使用申請單.pdf">
-        <span class="material-symbols-outlined">
-            download
-        </span>
-        先鋒13樓科學會辦使用申請單.pdf
-    </a>
-    <img src="/images/borrowing/step.png" alt="step.png" class="img-fluid">
+    <h1>申請表單</h1>
+    <div class="d-flex flex-column col-sm-6 h6">
+        <a href="https://forms.gle/5Vmr8hRtVtzjUm2P6" class="p-2">
+            <span class="material-symbols-outlined">
+                description
+            </span>
+            會辦借用申請表單
+        </a>
+        <a href="https://forms.gle/5cZgs2u9csavVMeq7" class="p-2">
+            <span class="material-symbols-outlined">
+                cloud_upload
+            </span>
+            照片回傳區
+        </a>
+        <a href="https://forms.gle/j53VhnepLysTY4jc7" class="p-2">
+            <span class="material-symbols-outlined">
+                check_box
+            </span>
+            教師簽名回傳區 (晚上時段限定)
+        </a>
+    </div>
+    <br />
+    <h1>社辦借用情況</h1>
+    <div id="interface">
+        <div class="calendar">
```

Comment:
Just curious, why do you choose using <div> over <table>? Is there some considerations with styling?